docs/codeql/ql-language-reference/annotations.rst

```diff
@@ -179,11 +179,11 @@ without ``external``, the compiler will report an error.
 ``final``
 =========
 
-**Available for**: |classes|, |member predicates|, |fields|
+**Available for**: |classes|, |aliases|, |member predicates|, |fields|
```

Comment:
What are these things in vertical bars? Links to something? There is a `type-aliases` anchor in `aliases.ast` if that's what you want. I don't know RST very well, but there's a link to it elsewhere that uses the syntax ``:ref:`type aliases <type-aliases>` ``.

Reply:
I've fixed the problem in the second commit.